src/commands/Dev/test/uploadConfig.test.ts

```diff
@@ -0,0 +1,97 @@
+import { describe, expect, jest, test } from '@jest/globals';
```

Comment:
I think there is no option of uploadConfig, but what is this test case for?

Reply:
I test for the config file, when I pass data to config.json, after calling devCommandHandle, the data that I passed have to exist in the config.json file.
![image](https://github.com/kintone-labs/kintone-cli/assets/117815925/0c6ddd8e-60d9-46dc-9ecb-6ee8ec7f77e3)